src/odemis/gui/cont/tabs/fibsem_tab.py

```diff
@@ -349,7 +354,20 @@ def _stop_streams_subscriber(self):
     def _start_streams_subscriber(self):
         self.tab_data_model.streams.subscribe(self._on_acquired_streams)
 
-    def on_dbl_click(self, evt):
+    def on_sem_stage_movement(self, evt):
+        active_canvas = evt.GetEventObject()
+        logging.debug(f"mouse down event, canvas: {active_canvas}")
+
+        if not active_canvas._tab_data_model.tool.value and self.pm.current_posture.value != SEM_IMAGING:
```

Comment:
The tab_data_model should be the same for all the controls in this tab, so you can just use `self.tab_data_model` (and hence, no need to call `GetEventObject()`.

Also, `tool.value` is more like an enumeration, so it's confusing to use `not`. I think it'd be clearer to write ` tool.value == TOOL_NONE`.